src/gsi/statsrad.f90

```diff
@@ -161,11 +161,11 @@ subroutine statsrad(aivals,stats,ndata)
 2011 format(8x,f16.8,8(i7,1x))
 2012 format(12x,A7,5x,8(a7,1x))
 2999 format(' Illegal satellite type ')
-1102 format(1x,i4,i5,1x,a16,2i7,1x,f10.3,1x,6(f11.7,1x))
+1102 format(1x,i4,i6,1x,a20,2i7,1x,f10.3,1x,6(f11.7,1x))
 1109 format(t5,'it',t13,'satellite',t23,'instrument',t40, &
           '# read',t53,'# keep',t65,'# assim',&
           t75,'penalty',t88,'qcpnlty',t104,'cpen',t115,'qccpen')
-1115 format('o-g',1x,i2.2,1x,'rad',2x,2A10,2x,3(i11,2x),4(g12.5,1x))
+1115 format('o-g',1x,i2.2,1x,'rad',2x,2A12,2x,3(i11,2x),4(g12.5,1x))
```

Comment:
Since `dplat` is length 11, I could see that `2A11` would be problematic (making so there is no space between `dplat` and `dtype`.  How about the following?

```suggestion
1115 format('o-g',1x,i2.2,1x,'rad',2x,A11,1x,A10,2x,3(i11,2x),4(g12.5,1x))
```